applications/advanced-concepts/zero-shot-translation.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Building and research
+title: Zero-shot translation
+description: Machine translation without parallel training data
+---
+
+**Zero-shot machine translation** is translation without parallel data between the source and target languages.
+Zero-shot translation is desirable because it can be too costly to create training data for each language pair.
+
+Zero-shot machine translation is an active area of research.
+Zero-shot models are typically lower quality than supervised translation models.
```

Comment:
One challenge here is that folks who don't know what zero-shot is probably don't know what supervision is.